Packs/Base/Scripts/GetIncidentsByQuery/GetIncidentsByQuery.py

```diff
@@ -91,22 +92,24 @@ def get_comma_sep_list(value):
 
 def main():
     # fetch query
-    query = build_incidents_query(demisto.args().get('query'),
-                                  demisto.args().get('incidentTypes'),
-                                  demisto.args()['timeField'],
-                                  demisto.args().get('fromDate'),
-                                  demisto.args().get('toDate'),
-                                  demisto.args().get('NonEmptyFields'))
-
-    incident_list = get_incidents(query, demisto.args()['timeField'],
-                                  int(demisto.args()['limit']),
-                                  demisto.args().get('fromDate'))
-    fields_to_populate = demisto.args().get('populateFields')
+    d_args = dict(demisto.args())
+    d_args['NonEmptyFields'] = d_args['NonEmptyFields'].replace('incident.', '')
```

Comment:
better using `lstrip()` function

Reply:
didn't know this function. 
If I understood it correctly, it might cause bugs as it receive as input a list of chars and not a word. 
So for example
```
In [3]: x = 'incident.info'

In [4]: x.lstrip('incident.')
Out[4]: 'fo'
```
Changed the logic so it'll be safer and replace only first prefix, it complicated a bit the solution